Sources/Shared/Core/VNError.swift

```diff
@@ -8,10 +8,14 @@
 
 import Foundation
 
-enum VimeoNetworkingError: Error {
+public enum VNError: Error {
     case encodingFailed(EncodingFailedReason)
+    case invalidURL
+    case requestError(Error)
+    case serializatingError
```

Comment:
Did you mean:
```suggestion
    case serializationError
```
or edit
```suggestion
    case serializingError
```
?